src/Assert.php

```diff
@@ -1249,8 +1338,10 @@ public static function alpha($value, $message = '')
     }
 
     /**
-     * @param mixed  $value
-     * @param string $message
+     * @psalm-pure
+     *
+     * @param string|int $value   can be an integer, since ctype_* functions accept ASCII integers -128 ~ +256
```

Comment:
While integers are allowed in the implementation i don't think we should 'show' that in the doc comment.

Reply:
The same goes for the other ctype based assertions as well

Reply:
985bf74a71a6df81eb2d01790b32d0617a69970a